packages/chopsticks/tsconfig.esm.json

```diff
@@ -0,0 +1,8 @@
+{
+	"extends": "./tsconfig.json",
+	"compilerOptions": {
+		"module": "ES2020",
```

Comment:
ES2021 because we use weakref
```suggestion
		"module": "ES2021",
```

Reply:
es2021 is not a valid module option, i will use es2022?